optaplanner-core/src/test/java/org/optaplanner/core/config/solver/SolverConfigMultiThreadedTest.java

```diff
@@ -22,22 +22,14 @@
 
 import org.junit.Ignore;
 import org.junit.Test;
-import org.optaplanner.core.api.solver.Solver;
-import org.optaplanner.core.api.solver.SolverFactory;
 import org.optaplanner.core.config.solver.testutil.MockThreadFactory;
 import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
 import org.optaplanner.core.impl.testdata.domain.TestdataSolution;
 import org.optaplanner.core.impl.testdata.domain.TestdataValue;
 import org.optaplanner.core.impl.testdata.util.PlannerTestUtils;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```

Comment:
I don't mind adhering to this one (I rarely look at the import statements). Any opinion on this @triceo and @rsynek ?

Do note that the KIE conventions had an exception for org.junit.Assert and org.mockito.Mockito:
  https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/1d35e1062cc735557a2be7177361e988ef728218/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml#L8

But the infamous "minimum common denominator" for Eclipse interoperability changed that too: https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/76aa61384524100db57968e674df9c140f7fa21f#diff-9d663bfb476dd8a86f4f213da123bfa5
It's a real shame that that PR got merged, not just because of this. (Ironically I was to one to actually pus the merge button...). Now that Eclipse is effectively dead (long live VSCode and Che), we might want to rethink that "minimum common denominator" approach.

Reply:
No opinion. I will use whatever we agree on and what can be configured in the IDE.

Reply:
Generally I try to avoid star imports, but I don't mind these two exceptions. If we configure our IDEs in this way, it's fine with me.

Reply:
I've add the code style discussion into this JIRA:
  https://issues.jboss.org/browse/PLANNER-1696
Please edit that jira to add any others we're missing.